src/ml/neural_net/mps_layer_helper.mm

```diff
@@ -5,10 +5,9 @@
  */
 
 #include <ml/neural_net/mps_layer_helper.h>
-#include <ml/neural_net/mps_weight.h>
-
 #import <ml/neural_net/mps_layer_conv_padding.h>
 #import <ml/neural_net/mps_layer_instance_norm_data_loader.h>
+#include <ml/neural_net/mps_weight.h>
```

Comment:
This looks like this should be (always have been) a #import, because it's an ObjC header. Since we're touching this line anyway, let's make it right!